### PR TITLE
Extra wide topper

### PR DESCRIPTION
**Pull request by gvonkoss**

### CODE REVIEW

File: server/controllers/article-helpers/topper-theme-map.js [hunk 1]
```diff
@@ -20,21 +21,40 @@ const getTopperSettings = (content, flags) => {
...
+			},
+			'extra': {
+				bgColour: 'slate',
+				layout: 'split-text-left',
+				template: 'themed'
+			},
+			'extra-wide': {
+				bgColour: 'slate',
+				layout: 'full-bleed-sliced', //TODO: Is this what we are calling this?
+				template: 'plp-wide'
```
Comment (adgad) [inline server/controllers/article-helpers/topper-theme-map.js]: I know _everyone_ is calling them PLPs but could we not? 😬

File: test/server/controllers/article-helpers/topper-theme-map.test.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+			}
+		}
+		const packageContentFixture = {
+			type: 'package',
+			design: {
+				theme: 'extra-wide'
+			}
+		}
+
+		it('applies slate offset topper if package theme is extra', () => {
```
Comment (adgad) [inline test/server/controllers/article-helpers/topper-theme-map.test.js]: I think this should also be true if the theme is extra-wide - should add test/code for that

File: server/controllers/article-helpers/topper-theme-map.js (hunk 1 above)
Comment (adgad) [inline server/controllers/article-helpers/topper-theme-map.js]: Actually we should just call the template full-bleed-sliced as well?

File: server/controllers/article-helpers/topper-theme-map.js [hunk 3]
```diff
@@ -8,33 +8,53 @@ const themeImageRatio = {
...
+				template: 'themed'
+			},
+			'extra': {
+				bgColour: 'slate',
+				layout: 'split-text-left',
+				template: 'themed'
+			},
+			'extra-wide': {
+				bgColour: 'slate',
+				layout: 'full-bleed-sliced', //TODO: Is this what we are calling this?
```
Comment (adgad) [inline server/controllers/article-helpers/topper-theme-map.js]: Shall we get rid of this TODO now?

File: server/controllers/article-helpers/topper-theme-map.js [hunk 4]
```diff
@@ -78,10 +98,8 @@ module.exports = (content, flags) => {
 	return Object.assign({},
 		topper,
 		{
-			images: content.leadImages,
-
 			headline: topper.headline || content.title,
-			standfirst: content.descriptionHTML || topper.standfirst || content.standfirst
+			standfirst: content.descriptionHTML || topper.standfirst || content.standfirst,
```
Comment (adgad) [inline server/controllers/article-helpers/topper-theme-map.js]: trailing comma

**Comment by gvonkoss** [comment]
:+1:

**Pull request merged**